packages/dd-trace/src/profiling/profilers/cpu.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+const dc = require('diagnostics_channel')
+
+const beforeCh = dc.channel('dd-trace:AsyncResourceStorage:before')
+const afterCh = dc.channel('dd-trace:AsyncResourceStorage:after')
+
+function getActiveSpan () {
+  const scope = global._ddtrace.scope()
```

Comment:
Why not use the `datadog-core` directly instead?

Reply:
I don't know what that is. 😅

Reply:
It hosts the new "scope manager" that is not coupled to tracing. It's available at `packages/datadog-core` and exports `{ storage }`. Then you could use it like this:

```js
const { storage } = require('../../../../datadog-core')
const store = storage.getStore()
const span = store.span
```